cmd/init/model.go

```diff
@@ -22,19 +22,20 @@ var (
 	sucessStyle  = styles.SucessStyle.Render
 	spinnerStyle = styles.SpinnerStyle
 	// helpStyle    = styles.HelpStyle.Render
-	errorStyle   = styles.ErrorStyle.Render
-	stages       = 4
```

Comment:
wont this give a linting error?

Reply:
Sorry, I did not check linting. I will fix it now.